exception.test.ts

```diff
@@ -0,0 +1,16 @@
+import { assertEquals } from "jsr:@std/assert";
+import { Try } from "./exception.ts";
+
+Deno.test("Try pick Error", () => {
+  const result = Try(() => {
+    throw new Error();
+  }).pick(Error).done(() => 42);
+  assertEquals(result, 42);
+});
+
+Deno.test("Try pick TypeError by Error predicate", () => {
```

Comment:
The description is inconsistent with what it tests. Could you fix either of them?